devtools-ci-index.yaml

```diff
@@ -2779,6 +2779,32 @@
             saas_git: saas-openshiftio
             timeout: '10m'
             extra_target: rhel
+        - '{ci_project}-{git_repo}':
+            git_organization: fabric8-services
+            git_repo: toolchain-operator
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '10m'
+        - '{ci_project}-{git_repo}-coverage':
+            git_organization: fabric8-services
+            git_repo: toolchain-operator
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_coverage.sh'
+            timeout: '30m'
+        - '{ci_project}-{git_repo}-build-master-coverage':
+            git_organization: fabric8-services
+            git_repo: toolchain-operator
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_coverage.sh'
+            timeout: '30m'
+        - '{ci_project}-{git_repo}-build-master':
+            git_organization: fabric8-services
+            git_repo: toolchain-operator
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
+            saas_git: saas-openshiftio
+            timeout: '10m'
+            extra_target: rhel
```

Comment:
this specifies RHEL image build, do you have everything for that?
I mean build scripts should recognize this and build image with rhel dockerfile etc

Reply:
ex: https://github.com/fabric8-services/fabric8-wit
take a look on build scripts for things like this https://github.com/fabric8-services/fabric8-wit/blob/master/cico_setup.sh#L158

Reply:
also please create ticket for request quay repos for this prj

Reply:
yes we do have for this repo, you can find it here https://github.com/fabric8-services/toolchain-operator/pull/12/files#diff-66536a55e28a60c7e24d2d4a43a8b89c

Reply:
> also please create ticket for request quay repos for this prj

where should I open a ticket?